docs/docs/testing-your-assistant.mdx

```diff
@@ -117,113 +117,138 @@ Here are some examples:
   </TabItem>
 </Tabs>
 
-By default Rasa Open Source saves test conversations to `tests/test_stories.yml`.
+Rasa Open Source looks for test conversations in all files with the prefix `test_`, e.g. `tests/test_stories.yml`.
 You can test your assistant against them by running:
 
 ```bash
 rasa test
 ```
 
-The command will always load all stories from any story files with filenames
-starting with `test_`, e.g. `test_stories.yml`. Your story test
-file names should always start with `test_` for this detection to work.
+See the [CLI documentation on rasa test command](./command-line-interface.mdx#rasa-test) for
+more configuration options.
 
 :::caution Testing Custom Actions
-[Custom Actions](./custom-actions.mdx) are **not executed as part of test conversations.** If your custom
+[Custom Actions](./custom-actions.mdx) are not executed as part of test conversations. If your custom
 actions append any events to the conversation, this has to be reflected in your test conversation
 (e.g. by adding `slot_was_set` events to your test conversation).
 
 To test the code of your custom actions, you should write unit tests
-for them and include these tests in your CI/CD pipeline.
+for them and include these tests in your [CI/CD pipeline](./setting-up-ci-cd.mdx).
 
 :::
 
-If you have any questions or problems, please share them with us in the dedicated
-[testing section on our forum](https://forum.rasa.com/tags/testing) !
-
 ## Evaluating an NLU Model
 
-:::tip hyperparameter tuning
-If you are looking to tune the hyperparameters of your NLU model,
-check out this [tutorial](https://blog.rasa.com/rasa-nlu-in-depth-part-3-hyperparameters/).
-:::
-
-
-A standard technique in machine learning is to keep some data separate as a test set.
-You can [split your NLU training data](./command-line-interface.mdx#create-a-train-test-split)
+In addition to end-to-end testing, you can also test the message processing model(NLU) separately.
+Once your assistant is deployed in the real world, it will be processing messages that it hasn't seen
+in the training data. Hence, you should always set aside some part of your training data for testing.
```

Comment:
in the training data. To simulate this, you should always set aside some part of your data for testing.